language/move-vm/runtime/src/data_cache.rs

```diff
@@ -189,11 +188,6 @@ impl<'r, 'l, C: RemoteCache> DataStore for TransactionDataCache<'r, 'l, C> {
                         None => {
                             let msg =
                                 format!("Failed to deserialize resource {} at {}!", ty_tag, addr);
-                            error!(
```

Comment:
callers of this function log themselves so there is no point in logging here and it would be a pin to hook up the logger at this level